docs/ref/template-response.txt

```diff
@@ -355,3 +355,74 @@ with a simple template and a context containing a queryset::
 
     def blog_index(request):
         return TemplateResponse(request, 'entry_list.html', {'entries': Entry.objects.all()})
+
+StreamingTemplateResponse and SimpleStreamingTemplateResponse
+=============================================================
+
+.. versionadded:: 1.9
+
+:class:`~django.template.response.StreamingTemplateResponse` and
+:class:`~django.template.response.SimpleStreamingTemplateResponse` allow the
+same features as :class:`~django.template.response.TemplateResponse` and
+:class:`~django.template.response.SimpleTemplateResponse` but stream the
+template instead of rendering it.
+
+SimpleStreamingTemplateResponse objects
+---------------------------------------
+
+.. class:: SimpleStreamingTemplateResponse()
+
+Attributes
```

Comment:
I don't think we need to document all the attributes and methods that are inherited from the parent.